tests/monitoring/vm_monitoring.go

```diff
@@ -304,3 +319,30 @@ var _ = Describe("[Serial][sig-monitoring]VM Monitoring", Serial, decorators.Sig
 		})
 	})
 })
+
+func createAgentVMI() *v1.VirtualMachineInstance {
+	virtClient := kubevirt.Client()
+	agentVMI := libvmifact.NewFedora(libnet.WithMasqueradeNetworking()...)
+
+	By("Starting a VirtualMachineInstance")
+	agentVMI, err := virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(agentVMI)).Create(context.Background(), agentVMI, metav1.CreateOptions{})
+	Expect(err).ToNot(HaveOccurred())
+	libwait.WaitForSuccessfulVMIStart(agentVMI)
+
+	getOptions := metav1.GetOptions{}
```

Comment:
nit: I'm not 100% sure, but I don't think we gain a lot by doing this, I think you can inline

Reply:
Done